l10n_it_vat_payability/migrations/14.0.1.0.0/pre-migrate.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2020 Sergio Zanchetta <https://github.com/primes2h>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.update_module_names(
+        env.cr,
+        [
+            ("l10n_it_esigibilita_iva", "l10n_it_vat_payability"),
+        ],
+    )
+    openupgrade.rename_xmlids(
```

Comment:
siamo sicuri che questo sia necessario? non dovrebbe essere sufficiente [update_module_names](https://github.com/OCA/openupgradelib/blob/973690cb02f43ce97d7cc9aaf6ffef2f7376c90d/openupgradelib/openupgrade.py#L1350-L1351)?

Reply:
Per il cambio di nome del modulo non è strettamente necessario. 
Ho solo colto l'occasione per adattare il nome dell'unica vista presente, per agevolare eventuali contributori esterni. 
Sebbene non sia specificato per le viste, anche le linee guida OCA dicono di usare la lingua inglese per nomi variabile e commenti.
https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst

Inoltre, in questo modo, possiamo dimostrare (ancor di più) con i fatti la ns. volontà di collaborazione e rispetto degli standard alla casa madre. No excuses! :stuck_out_tongue_winking_eye:

Reply:
Ok d'accordo, grazie per la delucidazione!